src/displayapp/screens/settings/Settings.cpp

```diff
@@ -49,35 +50,30 @@ std::unique_ptr<Screen> Settings::CreateScreen1() {
 }
 
 std::unique_ptr<Screen> Settings::CreateScreen2() {
-  std::array<Screens::List::Applications, 4> applications {{
-    {Symbols::shoe, "Steps", Apps::SettingSteps},
-    {Symbols::clock, "Set date", Apps::SettingSetDate},
-    {Symbols::clock, "Set time", Apps::SettingSetTime},
-    {Symbols::batteryHalf, "Battery", Apps::BatteryInfo}}};
+  std::array<Screens::List::Applications, 4> applications {{{Symbols::shoe, "Steps", Apps::SettingSteps},
+                                                            {Symbols::clock, "Set date", Apps::SettingSetDate},
+                                                            {Symbols::clock, "Set time", Apps::SettingSetTime},
+                                                            {Symbols::batteryHalf, "Battery", Apps::BatteryInfo}}};
```

Comment:
Inserting trailing commas to these arrays makes the format like it is in `CreateScreen1()`, which would be preferred.